components/application-service/base/kustomization.yaml

```diff
@@ -12,4 +12,19 @@ images:
 namespace: application-service
 
 patches:
-  - ./manager_resources_patch.yaml
+- ./manager_resources_patch.yaml
+- patch: |-
```

Comment:
I can be wrong, but you cannot combine the strategic merge patch and json patch in one. You need to put them in different sections:
```yaml
patchesStrategicMerge:
- ./manager_resources_patch.yaml

patches:
- patch: |-
```